scripts/foglamp

```diff
@@ -19,6 +19,19 @@
 set -e
 #set -x
 
+# RedHat specific
+platform=$([ `cat /etc/os-release | grep -c -i -E "centos|red hat"` -eq 0 ] && echo debian || echo redhat)
+if [[ $platform == "redhat" ]]
+then
+
+	export PATH=/opt/rh/rh-python36/root/usr/bin${PATH:+:${PATH}}
```

Comment:
These 5 steps are already done in /opt/rh/rh-python36/enable. Can also be done via "source scl_source enable rh-python36", rather than using absolute paths in "/opt/rh".

Reply:
the point is that when FogLAMP is started as a service we are using : 

sudo -u ${FOGLAMP_USER} "${FOGLAMP_ROOT}/bin/foglamp" start > /dev/null

and using a bourne - #!/bin/sh

and to solve this situation we have identify that solution with Mark.

Reply:
I identified the problem and solved - now using - source scl_source enable rh-python36